waspc/src/Wasp/Generator/FileDraft.hs

```diff
@@ -24,12 +26,14 @@ import Wasp.Generator.Templates (TemplatesDir)
 data FileDraft
   = FileDraftTemplateFd TmplFD.TemplateFileDraft
   | FileDraftCopyFd CopyFD.CopyFileDraft
+  | FileDraftCopyDirFd CopyDirFD.CopyDirFileDraft
```

Comment:
it's just a bit strange because so far all the file drafts were referring to a single file. and now we have a file draft that refers to a directory.

Reply:
Yes that is true. I am also not sure what to think about it. It makes sense and works ok, but it also does break some assumptions one might have based on the name FileDraft. But any other way would be pretty complicated right now I think, this is a simple and clean way to add this feature, except for naming. Maybe we can discuss it a bit, see what we can come up with. or just agree File can also mean Dir in this case or multiple Files :D.

Reply:
Agree, and will still just echo "everything is a file" to buy ourselves an out here for now :) I can add a TODO to revisit. 👍🏻

Reply:
Added the following comment above the definition:

```haskell
-- TODO: revisit the quick and dirty Linux interpretation of "everything is a file"
-- and treating a directory (`CopyDirFileDraft`) as a `FileDraft`. As is, this may be
-- a source of potential confusion and possibly tech debt to resolve later.
data FileDraft
```